volapi/chat.py

```diff
@@ -134,47 +139,51 @@ def timeout(self, duration=3600):
 
     @property
     def white(self):
-        return self.role is Roles.WHITE
+        return Roles.WHITE in self.role
 
     @property
     def user(self):
-        return self.role is Roles.USER
+        return Roles.USER in self.role
 
     @property
     def pro(self):
-        return self.role is Roles.PRO
+        return Roles.PRO in self.role
+
+    @property
+    def owner(self):
+        return Roles.OWNER in self.role
 
     @property
     def janitor(self):
-        return self.role is Roles.JANITOR
+        return Roles.JANITOR in self.role
 
     @property
     def donor(self):
-        return self.role is Roles.DONOR
+        return Roles.DONOR in self.role
 
     @property
     def green(self):
-        return self.pro or self.donor or self.user or self.janitor
+        return self.user
 
     @property
     def staff(self):
-        return self.role is Roles.STAFF
+        return Roles.STAFF in self.role
 
     @property
     def admin(self):
-        return self.role is Roles.ADMIN
+        return Roles.ADMIN in self.role
 
     @property
     def purple(self):
         return self.admin or self.staff
 
     @property
     def system(self):
-        return self.role is Roles.SYSTEM
+        return Roles.SYSTEM in self.role
 
     @property
     def logged_in(self):
-        return self.green or self.purple or self.janitor or self.pro
+        return self.green or self.purple
```

Comment:
Maybe to clear all this up a bit more and make green an opposite of purple this could be changed:
```python
    @property
    def green(self):
        return self.user and not self.purple
```
Then no moderator for example could be green and purple at the same time.

Reply:
That's what we get for not having comments on methods and having to come up with with meaning for them. Your proposal is good because it essentially replicates the user state in browser client so it would return true for people that have green color in chat, with only the trusted users being questionable but they shouldn't have green color in chat in the first place, so please commit your suggestion.